cornac/models/__init__.py

```diff
@@ -81,3 +81,12 @@
 from .vbpr import VBPR
 from .vmf import VMF
 from .wmf import WMF
+from .dmrl import recom_dmrl
```

Comment:
As a convention, we import the model class name directly instead of the module. Also, please place the import statement in the correct alphabetical order.